http_server.go

```diff
@@ -16,6 +17,8 @@ const (
 
 type NamespaceListerServer struct {
 	*http.Server
+	useTls  bool
```

Comment:
```suggestion
	useTLS  bool
```

IIRC, by convention this should be uppercase

Reply:
Fixed in f9ea3c6.